eahub/templates/eahub/privacy_policy.html

```diff
@@ -11,64 +11,181 @@
+<h3>Where we store your personal information</h3>
+<p>We store your personal information on secure servers. Our team members have access to this information in accordance with the level of access they need to fulfil their roles.</p>
 
-<h3>Do Not Track</h3>
-<p>Please note that we do not alter our Site’s data collection and use practices when we see a Do Not Track signal from your browser.</p>
+<h3>How long we keep it</h3>
+<p>We collect and store personal information for purposes connected with our business. As such, we will only retain your personal information for as long as necessary for those purposes.</p>
+<p>If you apply for a position at EA Hub, and your application is not successful, we will seek consent to keep your personal data (such as your email address, location, and resume) indefinitely for the purpose of future suitable job vacancies. If consent is not given, or withdrawn, we will keep your data for 12 months once the recruitment exercise ends.</p>
+<p>We regularly audit the personal information we hold and delete that which is not necessary.</p>
 
-<h3>Your Rights</h3>
-<p>If you are a European resident, you have the right to access personal information we hold about you and to ask that your personal information be corrected, updated, or deleted. If you would like to exercise this right, please contact us through the contact information below.</p>
+<h2 id="8-disclosing-your-personal-information">8. Disclosing your personal information</h2>
 
-<p>Additionally, if you are a European resident we note that we are processing your information in order to fulfill contracts we might have with you (for example if you make an order through the Site), or otherwise to pursue our legitimate business interests listed above.  Additionally, please note that your information will be transferred outside of Europe, including to Canada and the United States.</p>
+<h3>Newsletter subscriptions</h3>
+<p>If you sign up to one of our monthly newsletters, or register for an event we run, we will disclose your name and email address to Mailchimp, a service we use to manage the process of sending mass emails. You can opt out of a mailing list at any time by clicking the appropriate link in any email we send through Mailchimp.</p>
 
-<h3>Data Retention</h3>
-<p>When you enter information to the Site, we will maintain your Account Information and make it publicly available until you delete this information yourself or ask us to delete it for you.</p>
+<h3>Other disclosures</h3>
+<p>If we choose to sell, transfer, or merge parts of the company or our assets, we may disclose your personal information to the new owners. Alternatively, we may seek to merge with another company. If a change like that happens, then the new owners may use your personal information in the same way as set out in this policy.</p>
```

Comment:
The only thing that I'm not sure about is this paragraph. Above we say that we will not sell the data, but this statement means the opposite?

Reply:
There is a difference between selling the data and selling the company. I assume this is to cover cases where the company gets new owners or becomes part of another organisation. For this new entity to be able to run the Hub, they need access to the data

Reply:
I guess it's fine for me, but shouldn't we remove the "will not sell the data" sentence then? I don't really see a distinction between selling the company and selling the data from the users point of view?

Reply:
The difference is that if the company gets sold, the new owners presumably would continue running the Hub, so from the user's perspective not much changes and it's sort of in their interest, as they want the Hub to continue to exist. However, if we just sell the data to someone else, this is not in the users' interest as it doesn't support continue running the Hub.

Also, my default assumption here is that if it's fine for CEA (they have this verbatim in their policy, too), it should be fine for us.